src/etl/sars-cov-2/index.ts

```diff
@@ -0,0 +1,71 @@
+import Airtable, { FieldSet } from "airtable";
+import { pipe } from "fp-ts/lib/function.js";
+import {
+  asyncEtlStep,
+  etlStep,
+  getEnvironmentVariableOrThrow,
+  getMongoClient,
+  writeDataToMongoEtlStep,
+} from "../helpers.js";
+import { validateFieldSetFromAirtableStep } from "./steps/validate-field-set-from-airtable-step.js";
+import { cleanFieldNamesAndRemoveUnusedFieldsStep } from "./steps/clean-field-names-and-remove-unused-fields-step.js";
+import { parseDatesStep } from "./steps/parse-dates-step.js";
+import { latLngGenerationStep } from "./steps/lat-lng-generation-step.js";
+import { removeRecordsThatAreFlaggedNotToSave } from "./steps/remove-records-that-are-flagged-to-not-save-step.js";
+import { jitterPinLatLngStep } from "./steps/jitter-pin-lat-lng-step.js";
+import { transformIntoFormatForDatabaseStep } from "./steps/transform-into-format-for-database-step.js";
+import { filterStudiesThatDoNotMeetDataStructureRequirement } from "./steps/filter-studies-that-do-not-meet-data-structure-requirements.js";
+import { transformNotReportedValuesToUndefinedStep } from "./steps/transform-not-reported-values-to-undefined-step.js";
+
+const runEtlMain = async () => {
+  console.log("Running SarsCoV-2 ETL");
+  const airtableApiKey = getEnvironmentVariableOrThrow({
+    key: "AIRTABLE_API_KEY",
+  });
+  const airtableArboBaseId = getEnvironmentVariableOrThrow({
```

Comment:
```suggestion
  const airtableSC2BaseId = getEnvironmentVariableOrThrow({
```

Reply:
Fair lol. I'll change this on my next one.